config/internal/ml-metadata/metadata-grpc-tls-config-secret.yaml.tmpl

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ds-pipeline-metadata-grpc-tls-config-secret-{{.Name}}
+  namespace: {{.Namespace}}
+  labels:
+    component: metadata-grpc-server
+stringData:
+  config.proto: |
+    connection_config {
+      mysql {
+        host: "{{.DBConnection.Host}}"
+        port: {{.DBConnection.Port}}
+        database: "{{.DBConnection.DBName}}"
+        user: "{{.DBConnection.Username}}"
+        password: "{{.DBConnection.DecodedPassword}}"
+      }
+    }
```

Comment:
I see that there's still database config in the deployment [here](https://github.com/opendatahub-io/data-science-pipelines-operator/pull/683/files#diff-a4e0c9838f36c9cbebe2ac4004d445a4bc30e6957441ef1783766dd588d74a73R27-R31).

Remind me why we have to have the db config here too? Are the deployment config settings ignored if this config.proto file is used? (If the answer is yes, I'd be tempted to move to always sending in config.proto and removing the deployment db config, but that's a tech debt problem for another day.)

Reply:
Actually, this config is not even used. I had to add it because `connection_config` is mandatory. It doesn't make any difference if we set the values to `unused`, for example.